media-converter/src/audioconv.rs

```diff
@@ -217,7 +217,7 @@ struct NeedTranscodeHead {
 
 impl NeedTranscodeHead {
     fn new_from_caps(caps: &gst::CapsRef) -> Result<Self, gst::LoggableError> {
-        let s = caps.get_structure(0).ok_or(gst_loggable_error!(CAT, "Caps have no WMA data!"))?;
+        let s = caps.get_structure(0).ok_or_else(|| gst_loggable_error!(CAT, "Caps have no WMA data!"))?;
```

Comment:
without ok_or_else it would always run the function, no matter if that case happens or not. Thus taking memory and immediately dropping it again.